integrations/rails.md

```diff
@@ -95,7 +95,7 @@ This procedure creates a [basic Ruby on Rails application](https://guides.rubyon
     rails server
     ```
 
-The page containing the {{site.productname}} will be accessible at `http://localhost:<port>/` (default: [http://localhost:3000/](http://localhost:3000/)).
+The page containing the {{site.productname}} will be accessible at `http://localhost:<port>/` (default: http://localhost:3000/).
```

Comment:
Instead of removing this link, should we just make the link checker ignore it?

Reply:
Maybe? I would prefer not to hyperlink to localhost. Not sure why, but given it may or may not be there, not sure we should link there either. We should discuss this later?